swagger/public_v1/api_spec.json

```diff
@@ -1708,7 +1706,7 @@
             "in": "path",
             "required": true,
             "description": "The starting year of the recruitment cycle.",
-            "example": "2020",
+            "example": 2021,
```

Comment:
Should this be int Vs string?

Reply:
This has likely changed due to some changes made in the schema definition by some other commit. I'm not really sure but this chunk shouldn't be in this PR.